fixtures/user_profile_fixture.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
use copypaste in file to create user profiles for each user

Reply:
done